assets/ajax-cart.js.liquid

```diff
@@ -375,6 +408,7 @@ var ajaxCart = (function(module, $) {
 
       // If it has a data-line, update the cart
       if (line) {
+        $body.trigger('adjustCart.ajaxCart', [this, line, qty]);
```

Comment:
I was using it mainly for when the quantity in the cart was changed and then to easily get the id of the variant but that can easily be done using line for the position of the item in the cart and grabbing it from the input ... So I'll remove this, you're right it's just redundant and confusing... :)